components/Shell.tsx

```diff
@@ -309,20 +309,24 @@ function UserDropdown({ small }: { small?: boolean }) {
   const query = useMeQuery();
   const user = query.data;
 
-  return user ? (
```

Comment:
There are some edge cases where users can initiate a premium checkout, cancel it and they'll have a TRIAL username-less account. These changes will allow them to re-initialize the checkout on their dashboard OR choose a non-premium username (That check already exists in the dashboard).